### PR TITLE
The scripts I use to manage uploading to PyPI.

### PR DESCRIPTION
**Pull request by pathunstrom**

### CODE REVIEW

File: requirements-upload.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+certifi==2018.8.24
```
Comment (astronouth7303) [inline requirements-upload.txt]: Could you reduce this to what's actually required, not just a `pip freeze`?

Comment (pathunstrom) [reply requirements-upload.txt]: Probably doable. Looks to me like it's literally just twine and up to date pip.

**Comment by pathunstrom** [comment]
I think most of the clunky is that I'm intentionally adding manual checks to make sure it doesn't go terribly sideways.

**Pull request merged**